### ISSUE TITLE
Trim URL so that whitespace at the beginning won't reject the URL

### ISSUE THREAD

**Issue by gotonode** [UniversityOfHelsinkiCS/labtool#946]
The URL should be accepted even if it contains a space at the beginning or at the end. Just trim the string before checking its validity.
